### PR TITLE
Victoria Garcia - Ampers - JS Scrabble

### PR DESCRIPTION
**Pull request by Lasiorhine**
# JS Scrabble
Congratulations! You're submitting your assignment!

## Comprehension Questions
Question | Answer
:------------- | :-------------
What patterns were you able to use from your Ruby knowledge to apply to JavaScript? | Surprisingly few. Conditionals, rough method structure, loops and arrays -- but it feels like there was far more difference than similarity.
What was a challenge you faced in this assignment? | The tests, particularly, felt really impenetrable.
Do you have any recommendations on how we could improve this project for the next cohort? | Uuuh, not really, right now. Too bleary.

### CODE REVIEW

File: specs/scrabble.spec.js [hunk 1]
```diff
@@ -176,50 +176,59 @@ describe('Player', () => {
...
-
     });
   });
 
   describe('hasWon', () => {
-    test.skip('returns false when score < 100', () => {
-
-
+    test('returns false when score < 100', () => {
+      const player = new Scrabble.Player('joe');
```
Comment (CheezItMan) [inline specs/scrabble.spec.js:184]: It would work better to simply loop through verifying that `hashWon` returns false from a score of 0 to 99.  So you cover the edge cases.

File: specs/scrabble.spec.js [hunk 2]
```diff
@@ -176,50 +176,59 @@ describe('Player', () => {
...
+      player.plays = ['zz', 'qq'];
+      expect(player.hasWon()).toBe(false);
     });
 
-    test.skip('returns true when score == 100', () => {
-
-
+    test('returns true when score == 100', () => {
+      const player = new Scrabble.Player('jane');
```
Comment (CheezItMan) [inline specs/scrabble.spec.js:190]: You want to verify that hasWon is false at 99 and true at 100 and greater.

**Comment by CheezItMan** [comment]
# JS Scrabble
## What We're Looking For

Feature | Feedback
:------------- | :-------------
**Core Requirements** | 
Git hygiene | You need a few more commits.  Your last commit message had me chuckling.  
Comprehension questions | Check, hmmm.. what made the tests impenetrable? 
**General** | 
`score` calculates score, has appropriate params and return value |   Check
`highestScoreFrom` calculates highest scoring word, has appropriate params and return value | Check
**`Player` object** | 
Has `name` and `plays` properties | Check
Has `play`, `totalScore`, `hasWon` functions | Chekc
Has `highestScoringWord` and `highestWordScore` functions | Check
**Overall** | Well done, you hit all the learning goals for the project.  I'm sorry you had a tough time of it, but glad you stuck with it.
